modules/op-tkn-pipelines-compatibility-support-matrix.adoc

```diff
@@ -13,22 +13,20 @@ In the table, features are marked with the following statuses:
 TP:: Technology Preview
 GA:: General Availability
 
-// Writer, see http://dashboard.apps.cicd.ospqa.com/releases/componentmatrix/
-
 .Compatibility and support matrix
 [options="header"]
 |===
 
-| {pipelines-title} Version 8+| Component Version | OpenShift Version | Support Status
+| {pipelines-title} Version 10+| Component Version | OpenShift Version | Support Status
 
-| Operator | Pipelines | Triggers | CLI | Chains | Hub | {pac} | Results | Manual Approval Gate | |
+| Operator | Pipelines | Triggers | CLI | Chains | Hub | {pac} | Results | Manual Approval Gate | Pruner | Cache | |
```

Comment:
We might want to add a note here that console plugin follows the same version as OSP.
cc @anwesha-palit-redhat can you please confirm